Core/include/Acts/Surfaces/Surface.hpp

```diff
@@ -317,43 +318,33 @@ class Surface : public virtual GeometryObject,
                                                 const Vector3D& position,
                                                 const Vector3D& momentum) const;
 
-  /// Initialize the jacobian from local to global
-  /// the surface knows best, hence the calculation is done here.
-  /// The jacobian is assumed to be initialised, so only the
-  /// relevant entries are filled
+  /// Calculate the jacobian from local to global which the surface knows best,
+  /// hence the calculation is done here
   ///
   /// @todo this mixes track parameterisation and geometry
   /// should move to :
   /// "Acts/EventData/detail/coordinate_transformations.hpp"
   ///
   /// @param gctx The current geometry context object, e.g. alignment
-  /// @param jacobian is the jacobian to be initialized
-  /// @param position is the global position of the parameters
-  /// @param direction is the direction at of the parameters
-  /// @param pars is the parameter vector
-  virtual void initJacobianToGlobal(const GeometryContext& gctx,
-                                    BoundToFreeMatrix& jacobian,
-                                    const Vector3D& position,
-                                    const Vector3D& direction,
-                                    const BoundVector& pars) const;
-
-  /// Initialize the jacobian from global to local
-  /// the surface knows best, hence the calculation is done here.
-  /// The jacobian is assumed to be initialised, so only the
-  /// relevant entries are filled
+  /// @param boundParams is the bound parameters vector
+  ///
+  /// @return Jacobian from local to global
```

Comment:
I would suggest to describe why method takes the bound parameters as input. Since we stumbled over this during our discussion, it would be good if we do not forget about this in the future.

Reply:
Yeah, I add some description there.